api/v1/routes/blog.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any, Dict
+from fastapi import APIRouter, Depends, HTTPException, Query, status
+from sqlalchemy.orm import Session
+from sqlalchemy.exc import SQLAlchemyError
+from api.v1.models.blog import Blog
+from api.db.database import get_db
+import logging
+
+blog = APIRouter(prefix="/blogs", tags=["Blog"])
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+@blog.get("", status_code=status.HTTP_200_OK)
+def list_blog(
+    page: int = Query(1, ge=1),
+    page_size: int = Query(10, ge=1, le=100),
+    db: Session = Depends(get_db),
+) -> Dict[str, Any]:
+    try:
+        offset = (page - 1) * page_size
+        query = (
+            db.query(Blog)
+            .filter(Blog.is_deleted == False)
+            .order_by(Blog.created_at.desc())
+        )
+        total_count = query.count()
+        blogs = query.offset(offset).limit(page_size).all()
+
+        next_page = None
+        if offset + page_size < total_count:
+            next_page = f"/api/v1/blogs?page={page + 1}&page_size={page_size}"
+
+        prev_page = None
+        if page > 1:
+            prev_page = f"/api/v1/blogs?page={page - 1}&page_size={page_size}"
+
+        results = []
+        for blog in blogs:
+            blog_dict = {
+                "id": blog.id,
+                "title": blog.title,
+                "excerpt": blog.excerpt,
+                "image_url": blog.image_url,
+                "tags": blog.tags,
+                "created_at": blog.created_at,
+            }
+            results.append(blog_dict)
+
+        return {
+            "count": total_count,
+            "next": next_page,
+            "previous": prev_page,
+            "results": results,
+        }
+
```

Comment:
Use `JsonResponseDict` class in the `utils.json_response` module for your response, for standardizing the response according to the convention

Reply:
I have made the required changes. Check if it conforms to what you expected.

Reply:
Let me check the failing tests

Reply:
JsonResponseDict for standardized API responses has been integrated.